### PR TITLE
Categories page

### PR DESCRIPTION
**Pull request by dipen2000**
I created this categories page please review my PR and give me feedback on that.

### CODE REVIEW

File: CSS/styles.css [hunk 1]
```diff
@@ -49,7 +49,7 @@
 }
 
 .container{
-    border : 3px solid red;
+    /* border : 3px solid red; */
```
Comment (pnchinmay) [inline CSS/styles.css:52]: remove these comments

File: CSS/styles.css [hunk 2]
```diff
@@ -58,14 +58,15 @@
 header{
     font-family: var(--heading-font);
     font-size : 3rem;
+    font-weight : bold;
 }
 
 main{
-    border : 3px solid red;
+    /* border : 3px solid red; */
```
Comment (pnchinmay) [inline CSS/styles.css:65]: same here

**Review by pnchinmay — APPROVED** [review]

**Comment by shruthipatil11** [comment]
.rules-container{
    border : 3px solid yellow; 

Hex codes are preferred than the  color name directly

File: CSS/styles.css [hunk 3]
```diff
@@ -58,14 +58,15 @@
 header{
     font-family: var(--heading-font);
     font-size : 3rem;
+    font-weight : bold;
```
Comment (dev-ashishsethi) [inline CSS/styles.css:61]: you could have also used <strong> tag

File: CSS/styles.css [hunk 4]
```diff
@@ -77,17 +78,48 @@ ol{
...
+}
 
+.single-category-container{
+    /* border : 3px solid green; */
+    display : flex;
+    flex-direction: column;
+    padding : 1.2rem;
+    width : 35rem;
+    gap : 10px;
+    border-radius: 0.8rem;
```
Comment (dev-ashishsethi) [inline CSS/styles.css:103]: when measurement is less than 1 rem then use pixels

**Review by dev-ashishsethi — APPROVED** [review]

**Review by j24m — APPROVED** [review]
Hello other people have already suggested some changes, like removing the comments, you must do that before merging, rest of the code is fine, class names are readable and PRs are small , keep up the great work:)

**Comment by shruthipatil11** [comment]
looks good to merge

File: CSS/styles.css [hunk 5]
```diff
@@ -77,17 +78,48 @@ ol{
...
+    background-color : var(--light-test-color);
+    cursor : pointer;
+    /* height : 25rem; */
+}
 
+.img-container{
+    /* border : 3px solid purple; */
+    width : 100%;
+    height : 20rem;
```
Comment (rahulyadav139) [inline CSS/styles.css:112]: The files have way more than redundant comments. If you are not planning to use these in any way you should remove these. Other than that everything is great. Great work.

**Review by rahulyadav139 — APPROVED** [review]

**Pull request merged**